cli/src/pcluster/models/cluster_config.py

```diff
@@ -1031,6 +1031,20 @@ def is_cw_logging_enabled(self):
             else False
         )
 
+    @property
+    def is_cw_dashboard_enabled(self):
+        """Return True if CloudWatch Dashbboard is enabled."""
+        return (
+            self.monitoring.dashboards.cloud_watch.enabled
+            if self.monitoring and self.monitoring.dashboards and self.monitoring.dashboards.cloud_watch
+            else False
```

Comment:
The default should be True, by default CW Dashboard is enabled.

Let's add a constant in contants.py module and use it here and in the Schema:
```
CW_DASHBOARD_ENABLED_DEFAULT = True
```
```
class CloudWatchDashboards(Resource):
...
        self.enabled = Resource.init_param(enabled, default=CW_DASHBOARD_ENABLED_DEFAULT)
```

Reply:
The above you mentioned are changed. I didn't change it in the config as the following. Because it will break the unit test for batch and slurm now. On the second thought, I think return `else True` or `else False` will be the same. Because we already set the default value of `self.monitoring.dashboards.cloud_watch.enabled`=True. Self.monitoring also need to have a default other than None. 
@property
def is_cw_dashboard_enabled(self):
    """Return True if CloudWatch Dashbboard is enabled."""
    return (
        self.monitoring.dashboards.cloud_watch.enabled
        if self.monitoring and self.monitoring.dashboards and self.monitoring.dashboards.cloud_watch
        else CW_DASHBOARD_ENABLED_DEFAULT
    )